src/components/spreadsheet/utils/sensi-properties.ts

```diff
@@ -0,0 +1,32 @@
+import { ICellRendererParams } from 'ag-grid-community';
+
+interface LinesType {
+    [key: string]: string;
+}
+export class SensiProperties {
```

Comment:
Did you get part or whole of this code from the AGGrid documentation ? If yes, can you add the link to the page ?
It's usefull if one day the code brake.